sklearn/gaussian_process/kernels.py

```diff
@@ -763,10 +784,22 @@ def __repr__(self):
 
 
 class Product(KernelOperator):
-    """Product-kernel k1 * k2 of two kernels k1 and k2.
+    """The `Product` kernel takes two kernels :math:`k_1` and :math:`k_2`
+    and combines them via
+
+    .. math::
+        k_{prod}(X, Y) = k_1(X, Y) * k_2(X, Y)
+
+    Using the Product-kernel is equivalent to multiplying two kernels::
```

Comment:
Please apply the same changes here and in the Exponential Kernel too

Reply:
Done